lib/jekyll/utils.rb

```diff
@@ -313,6 +313,20 @@ def merged_file_read_opts(site, opts)
       merged
     end
 
+    # Safely load YAML strings
+    def safe_load_yaml(yaml)
+      Psych.safe_load(yaml, :permitted_classes => [Date, Time], aliases: true)
```

Comment:
I am wondering if it would make sense to add `Symbol` to the list of permitted classes. Indeed, with this patch in Debian, there is a bug open because of failing tests in jekyll asciidoc plugin

https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1030465

caused by the fact that Symbol is not allowed to be loaded by Psych.